plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConnector.java

```diff
@@ -193,7 +192,6 @@ public ConnectorAccessControl getAccessControl()
     public ConnectorTransactionHandle beginTransaction(IsolationLevel isolationLevel, boolean readOnly, boolean autoCommit)
     {
         checkConnectorSupports(READ_COMMITTED, isolationLevel);
-        verify(autoCommit, "Catalog only supports writes using autocommit: DeltaLake");
```

Comment:
Should this be moved somewhere, or do non-autocommit writes to Delta just never get this far?

Reply:
from cmt msg

> The commit removes the check from `DeltaLakeConnector` since appropriate
check is being done in `InMemoryTransactionManager`.